### ISSUE TITLE
Support for Xiaomi Vacuum-mop pro (STYTJ02YM viomi.vacuum.v9)

### ISSUE THREAD

**Issue by dumashsu** [rytilahti/python-miio#1334]
Device information:

    Name(s) of the device: Xiaomi Vacuum-Mop pro

Use miiocli device --ip <ip address> --token <token> info

    Model: viomi.vacuum.v9
    Hardware version: Linux
    Firmware version: 3.5.8_0020

**Comment by rytilahti** [comment]
Is the device working just fine using `miiocli viomivacuum`? If yes, v9 should just be added to the list of supported devices (https://github.com/rytilahti/python-miio/blob/master/miio/integrations/vacuum/viomi/viomivacuum.py#L65).

**Comment by dumashsu** [comment]
sorry! new device not bug

**Comment by GohanDS** [comment]
Yes, add support for Xiaomi Vacuum-mop pro please. Already it doesnt work.
